import System.Exit

main = exitFailure
